# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../emacs-xemacs-shared/no-ce-tags.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xemacs-21.4.12/lib-src/Makefile.in.in.orig	2003-05-19 23:44:57.000000000 +0200
+++ xemacs-21.4.12/lib-src/Makefile.in.in	2003-05-19 23:52:53.000000000 +0200
@@ -91,7 +91,7 @@
 #ifdef HAVE_MS_WINDOWS
  winclient\
 #endif
- etags ctags b2m ootags
+ b2m ootags
 
 PUBLIC_INSTALLABLE_SCRIPTS=\
 #ifdef INSTALL_GNUSERV
